include/Swr_Model.cpp

```diff
@@ -299,9 +299,11 @@ void Swr_Model::write_Xml(TiXmlElement *parent, const uint8_t *buf, size_t size)
 	TiXmlElement* node_models = new TiXmlElement("Models");
 	parent->LinkEndChild(node_models);
 
+  size_t i = 0;
+  nbModels = 1;
```

Comment:
Dirty hack, should be removed before merge - but we need a way to split models more easily.